source/includes/app-services-cli-overview.rst

```diff
@@ -158,3 +158,69 @@ The CLI stores identifiers and configuration metadata in the
    * - | ``client_app_id``
        | ``string``
      - The human-readable Client App ID.
+
+.. _cli-profiles:
+
+CLI Profiles
+------------
+
+The CLI stores information about its users in a profile. This lets it
+remember information between commands. For example, when you log in with
+an Atlas Admin API Key, the CLI stores the API Key and the current
+session access token. Then it reuses that token for subsequent commands
+until it expires.
+
+You can set up multiple named profiles and choose a profile to use for
+any given CLI command. If you don't specify one, the CLI uses the
+**default profile**, which is just a regular profile named ``default``.
```

Comment:
Nit: the use of "regular" here implies it's possible to have other types of profiles that aren't "regular." Maybe we could simplify?
```suggestion
**default profile**, which is a profile named ``default``.
```